src/api/ApiClient.ts

```diff
@@ -60,16 +60,16 @@ class ApiClient {
   request = async (
     config: AxiosRequestConfigWithRetries & { retries?: number }
   ) => {
-    this.debug(
```

Comment:
Commented these out for now, but if you want to see the logs of every request in console, then feel free to comment them back in. For me they started getting a bit in the way during these tasks